src/ol_infrastructure/applications/jupyterhub/Pulumi.applications.jupyterhub.CI.yaml

```diff
@@ -2,8 +2,10 @@
 secretsprovider: awskms://alias/infrastructure-secrets-ci
 encryptedkey: AQICAHi3MZ/Pjy2dahB1Qm+zKkKDPV1b9MYPGp7k649HPjmOHAFqnCOCAZSgyF1ZWf0UadNJAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMo8Vlyuwrp1ocOIPzAgEQgDtuHDo3PpUPnI8Ld39Bq5dR9m0mnxpqjeqfAWopoZssGhgjoG9CfYrqTNsw4qr8fHcnvtAHzJoKzyDD4w==
 config:
-  jupyterhub:binderhub_domain: "binder.ci.learn.mit.edu"
+  binderhub:domain: "binder.ci.learn.mit.edu"
   jupyterhub:target_vpc: "applications_vpc"
+  jupyterhub:rds_password: # This MUST be changed, it's reusing the shared PW
```

Comment:
This can safely be randomly generated for every stack. We don't actually care what the password is, and for break-glass we can decrypt it from this stack. I like to use the command `python -c "import secrets; print(secrets.token_urlsafe())"` to generate passwords.